.vscode/settings.json

```diff
@@ -8,5 +8,8 @@
     }
   ],
   "editor.formatOnSave": false,
-  "eslint.autoFixOnSave": true
+  "eslint.autoFixOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

Comment:
Unrelated changes?

Reply:
that came with a git pull origin 174 ....

Reply:
I had the same thing – it's an `ESLint` plugin change that came with the latest `VSCode` update. The question is: Do we want this setting activated for all developers? Or should we go the other direction and add `.vscode/settings.json` to `.gitignore`?